controllers/provisioners/eks/scaling/launchtemplate.go

```diff
@@ -243,6 +246,41 @@ func (lt *LaunchTemplate) Drifted(input *CreateConfigurationInput) bool {
 		drift = true
 	}
 
+	if len(latestVersion.LaunchTemplateData.LicenseSpecifications) != len(input.LicenseSpecifications) {
+		log.Info("detected drift", "reason", "Number of LicenseSpecifications has changed", "instancegroup", lt.OwnerName)
+		drift = true
+	}
+	for _, v := range latestVersion.LaunchTemplateData.LicenseSpecifications {
+		if !contains(input.LicenseSpecifications, aws.StringValue(v.LicenseConfigurationArn)) {
+			log.Info("detected drift", "reason", "LicenseSpecifications has changed", "instancegroup", lt.OwnerName,
+				"previousValue", aws.StringValue(v.LicenseConfigurationArn),
+				"newValue", input.LicenseSpecifications,
+			)
+			drift = true
+		}
+	}
+
+	if input.Placement == nil {
```

Comment:
Same, we should have a function to get a `LaunchTemplatePlacement` from `input.Placement`  in the AWS pkg which does this nil check as well.
Then all we need is sorting and reflect.DeepEqual()